content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -43,26 +45,21 @@ Keep in mind that setting up your cluster to use cloud controller manager will c
 * cloud information about nodes in the cluster will no longer be retrieved using local metadata, but instead all API calls to retrieve node information will go through cloud controller manager. This may mean you can restrict access to your cloud API on the kubelets for better security. For larger clusters you may want to consider if cloud controller manager will hit rate limits since it is now responsible for almost all API calls to your cloud from within the cluster.
 
 
-As of v1.8, cloud controller manager can implement:
+The cloud controller manager can implement:
 
-* node controller - responsible for updating kubernetes nodes using cloud APIs and deleting kubernetes nodes that were deleted on your cloud.
-* service controller - responsible for loadbalancers on your cloud against services of type LoadBalancer.
-* route controller - responsible for setting up network routes on your cloud
+* Node controller - responsible for updating kubernetes nodes using cloud APIs and deleting kubernetes nodes that were deleted on your cloud.
+* Service controller - responsible for loadbalancers on your cloud against services of type LoadBalancer.
```

Comment:
nit: Check the other pages for this list of controllers (if listed here, should be the same content?). Should the content be repeated or listed once in the concept section?

Reply:
Maybe that's OK to save for a different PR? There are already a lot of changes in this one.